src/main/scala/viper/gobra/frontend/Desugar.scala

```diff
@@ -2261,11 +2261,11 @@ object Desugar {
       }
     }
 
-    lazy val interfaceImplementations: Map[in.InterfaceT, Set[in.Type]] = {
+    lazy val interfaceImplementations: Map[in.InterfaceT, SortedSet[in.Type]] = {
       info.interfaceImplementations.map{ case (itfT, implTs) =>
         (
           interfaceType(typeD(itfT, Addressability.Exclusive)(Source.Parser.Unsourced)).get,
-          implTs.map(implT => typeD(implT, Addressability.Exclusive)(Source.Parser.Unsourced))
+          SortedSet(implTs.map(implT => typeD(implT, Addressability.Exclusive)(Source.Parser.Unsourced)).toSeq: _*)
```

Comment:
Is this how a data structure is changed into a sorted set. I would have expected some function that can take an Iterable directly.